frontend/style.css

```diff
@@ -16,132 +18,211 @@ html, body {
 /* Radiosity menu */
 #menu {
   position: absolute;
-  top: 15px;
-  left: 15px;
+  top: 1em;
```

Comment:
Thank for having replaced all my px values by em.

Reply:
you'll get used to avoiding `px`, I hope 8-)